src/test/util.cc

```diff
@@ -182,15 +182,15 @@ void ProcessSQLs(sdk::SQLRouter* sr, std::initializer_list<absl::string_view> sq
     hybridse::sdk::Status status;
     for (auto sql : sqls) {
         sr->ExecuteSQL(std::string(sql), &status);
-        EXPECT_TRUE(status.IsOK()) << "running sql=" << sql << " failed: code=" << status.code << ", msg=" << status.msg
+        ASSERT_TRUE(status.IsOK()) << "running sql=" << sql << " failed: code=" << status.code << ", msg=" << status.msg
```

Comment:
we need `EXPECT*`, some context require all SQLs executed even failure

Reply:
OK, but it's not a good idea to expect when init(e.g. create db/table), if create failed, the log will be too long and may confuse us.